README.md

```diff
@@ -88,7 +88,9 @@ server.plugins['hapi-sequelizejs'][opts.name] = new DB(opts.sequelize, models);
 
 ### API
 
-#### `getDb(name)`
+#### Using `request` object
+
+#####`getDb(name)`
```

Comment:
This broke the markdown, please add a space after the `#####`

Reply:
Fixed in 898c7885954ff24f67da4c35eb157e040fe37d3d